src/web/components/SlotBodyEnd.tsx

```diff
@@ -1,5 +1,8 @@
 import React, { useEffect, useState, useRef } from 'react';
+import * as emotion from 'emotion';
 import { css } from 'emotion';
+import * as emotionCore from '@emotion/core';
+import * as emotionTheming from 'emotion-theming';
```

Comment:
Important to note here that the `sendOphanReminderEvent` function below needs to be assigned to the `onReminderOpen` prop in the Epic.